UnboundLib/GameModes/GameModeHandler.cs

```diff
@@ -57,6 +60,74 @@ public virtual void ChangeSetting(string name, object value)
 
         public abstract void PlayerDied(Player killedPlayer, int playersAlive);
 
+        public virtual void OnPlayerLeftRoom(Photon.Realtime.Player otherPlayer)
+        {
+            List<Player> disconnected = PlayerManager.instance.players.Where(p => p.data.view.ControllerActorNr == otherPlayer.ActorNumber).ToList();
+
+            int playersAlive = PlayerManager.instance.players.Except(disconnected).Where(p => !p.data.dead).Count();
+
+            foreach (Player player in disconnected)
+            {
+                try
+                {
+                    this.PlayerDied(player, playersAlive);
+                }
+                catch { }
+            }
```

Comment:
Should we check if the player is already dead?

Reply:
> Should we check if the player is already dead first?

yes, good call